src/styles/card.css

```diff
@@ -0,0 +1,117 @@
+.card {
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`